react/typings/vtex.responsive-values.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module 'vtex.responsive-values' {
```

Comment:
Not sure if you need this. Try running `vtex setup --typings` to see if these types are needed or not

Reply:
I ended up needing it. Running the setup command on `vtex.responsive-values` only gave me an empty `node_modules` folder, no typings.